library/common/extensions/filter/platform_filter.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#include "envoy/http/filter.h"
+
+#include "extensions/filters/http/common/pass_through_filter.h"
+
+#include "library/common/extensions/filter/c_types.h"
+
+namespace Envoy {
+namespace Http {
+/**
+ * Harness to bridge Envoy filter invocations up to the platform layer.
+ */
+class PlatformExtensionFilter final : public PassThroughFilter {
```

Comment:
nit: above in c_types this was called `PlatformHarnessFilter`

Reply:
Yeah, I renamed a few times and missed that one. Settled on PlatformExtensionFilter, as in "the filter that enables platform extensions".